parachains/runtimes/assets/statemint/src/xcm_config.rs

```diff
@@ -166,85 +166,82 @@ impl Contains<RuntimeCall> for SafeCallFilter {
 			}
 		}
 
-		match call {
+		matches!(
```

Comment:
Are you sure about that one?
Using the `matches!` macro we lose the completeness check.
[Color](https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=6673dbf7f2227c8b1e6059051b0e371b)